docs/en/user_guides/1_config.md

```diff
@@ -103,15 +103,15 @@ which is convenient to conduct various experiments.
           consensus=dict(type='AvgConsensus', dim=1),  # Config of consensus module
           dropout_ratio=0.4,  # Probability in dropout layer
           init_std=0.01, # Std value for linear layer initiation
-          average_clips=None),
+          average_clips=None),  # Method to average multiple clip results
```

Comment:
assign `average_clips` as `prob` is better?

Reply:
done